app/controllers/my_facilities/drug_stocks_controller.rb

```diff
@@ -37,8 +37,10 @@ def create
 
   def drug_report
     @facilities = filter_facilities
+      .where.not(facility_size: :community)
```

Comment:
Should this move into the drug stocks query, so that it's captured no matter where it is run from?

Reply:
Don't think so. The query should just operate on the data its passed, any filtering should happen at the call site.

Reply:
Why? Is there a case where it makes sense to report on community facilities as well?

Reply:
It totally might end up that way, there's a degree of uncertainty about how this feature will end up working. 
Regardless, it shouldn't be the QueryObject's responsibility to do filtering when we have a concern dedicated to doing this.

Reply:
I'm not sure about the difference in concerns here, but the fact that we filter these to show on the filter anyway overrides the perspective I had. The way you've written it makes sense from a reuse angle as well. 👍